frontend/src/Page.js

```diff
@@ -7,26 +7,46 @@ import CreateEmployee from './employees/CreateEmployee';
 import UpdateEmployee from './employees/UpdateEmployee';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './style.css';
-
+import LogOut from './LogOut';
 
```

Comment:
importok is legyenek egyben, nem kell az üres sor, illetve lehet őket úgy rendezni/szeparálni, hogy minden third-party az elején és a saját fájlok utána

Reply:
Oké, most már szépek az importok.